src/client/Client.py

```diff
@@ -0,0 +1,60 @@
+"""Frontend application entry point."""
```

Comment:
wouldn't it be better to keep modular structure of files?
i.e. 
clientside/client/client.py
clientside/client/__init__.py
clientside/UserInterface/main.py
clientside/UserInterface/__init__.py
clientside/UserInterface/window.py
etc?